bash_103_combining_commands/bash_103_combining_commands.md

```diff
@@ -241,7 +256,8 @@ sort Animals.csv >> animals.txt
 
 Again there is no output shown because it was redirected into the file `animals.txt`. Check that `animals.txt` now contains the word count at the top, followed by a sorted copy of the data from `Animals.csv`.
 
-<div class = "learn-more">
+<div class = "behind-the-scenes">
```

Comment:
I feel like with some slight tweaking, this actually could be a perfectly acceptable learn more, since we're linking out? see my suggestion below